bin/parse_pipe.awk

```diff
@@ -0,0 +1,30 @@
+#! /usr/bin/awk -f
+
```

Comment:
What is this script supposed to do?

Reply:
This script is a helper script for viz_pipe.